src/components/index/page-sections/TestimonialSection.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import TestimonialPhoto from '../../../images/testimonial_profile.jpeg';
+import TestimonialLogo from '../../../images/flink-logo.png';
+import TestimonialSectionContentContainer from './TestimonialSectionContentContainer';
+
+const TestimonialSection = () => {
+  return (
+    // Section Container
+    <div className="container mx-auto mb-40">
+      {/* 2 COL GRID @largerScreen */}
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-10">
+        <TestimonialSectionContentContainer
+          personasImage={TestimonialPhoto}
```

Comment:
würde jeweils statt 'personas' -> 'persona' nehmen in den props für die Sections hier. Hattest ja in Slack gefragt wegen den props über mehrere Levels, denke hier würde sich ein TestimonalSectionContext dann schon anbieten.